src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -659,6 +659,10 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
         $container->setParameter('translator.logging', $config['logging']);
 
+        if (isset($config['cache'])) {
```

Comment:
You should prefix cache entries with something specific with the current installation to prevent conflicts if several Symfony apps run on the same server.

The example for the validator: https://github.com/aitboudad/symfony/blob/translation_catalogue_cache/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L774